Kajabi/sage-lib
# [SAGE_1147] Creation Wizard Product Selection

## Description
Adds the creation wizard to the mocks page and the product selection portion.


## Screenshots
![Screen Shot 2022-01-12 at 9 03 02 AM](https://user-images.githubusercontent.com/1175111/149187047-8aaa84e2-ff38-497d-aa8b-3d32170116be.png)

## Testing in `sage-lib`

- Navigate to http://localhost:4000/pages/mocks
- Click on the creation wizard option
- Verify product selection looks and works as expected.


## Testing in `kajabi-products`
N/A


## Related
https://github.com/Kajabi/sage-lib/issues/1147
